scripts/generator/graphql_generator/csharp/SDK/iOS/iOSNativeCheckout.ApplePayEventReceiver.cs.erb

```diff
@@ -80,16 +149,67 @@ namespace <%= namespace %>.SDK.iOS {
             }
         }
 
+         private ApplePayAuthorizationStatus GetAuthorizationStatusFromCheckoutUserErrors(List<UserError> errors) {
+
+            ApplePayAuthorizationStatus statusToReturn = ApplePayAuthorizationStatus.Failure;
+
+            foreach (var error in errors) {
+                var fields = error.field();
+
+                // If there exists a UserErrorPath that is atleast 2 Fields long, and the second last field is `billingAddress`
```

Comment:
I suppose this could break if graphQL decides to add another branching field to `billingAddress`. To be more solid I suppose we can check all the fields for a matching string